test/tests.js

```diff
@@ -23,7 +23,6 @@
   });
 
   test('wrap a DOM element with $', 1, function () {
-
     ok($(btn) instanceof $);
```

Comment:
Do we really need `$(...) instance of $`?

If we don't need to ensure `instance of` works, we can shave off a few bytes by simplifying line 27:
`i[d] = $.fn = c`

I can see the appeal to keep it though :)

Reply:
well i mean, we could not, but i think it makes the interface more robust, a part from other reasons, mostly dx and good api design, being able to know anything wrapped in $(...) is an instance of ki ($).

originally i would think that because we are basically wrapping the native dom apis in a way, differentiating this carries a meaning

Reply:
I have another PR that fixes this if you want :) Let's first try to merge this one!

I agree with you, I think it's quite important to not confuse people.